File: SymmetricNumber.java
public class MirrorNumberPyramid {
    public static void main(String[] args) {
        int rows = 4;

        for (int i = 1; i <= rows; i++) {
            // Left side
            for (int j = 1; j <= i; j++)
                System.out.print(j + " ");

            // Spaces
            for (int j = 1; j <= 2 * (rows - i); j++)
                System.out.print("  ");

            // Right side
            for (int j = i; j >= 1; j--)
                System.out.print(j + " ");

            System.out.println();
        }
    }
}

File: SymmetricNumber.py
rows = 4
for i in range(1, rows + 1):
    # Left side increasing numbers
    for j in range(1, i + 1):
        print(j, end=" ")
    
    # Spaces in the middle
    print("  " * (rows - i), end="")
    print("  " * (rows - i), end="")
    
    # Right side decreasing numbers
    for j in range(i, 0, -1):
        print(j, end=" ")
    
    print()
